firefox-ios/firefox-ios-tests/Tests/XCUITests/PrivateBrowsingTest.swift

```diff
@@ -230,6 +230,45 @@ class PrivateBrowsingTest: BaseTestCase {
         XCTAssertTrue(app.buttons["Copy Link"].exists, "The option is not shown")
         XCTAssertTrue(app.buttons["Download Link"].exists, "The option is not shown")
     }
+
+    // https://testrail.stage.mozaws.net/index.php?/cases/view/2497357
+    func testAllPrivateTabsRestore() {
```

Comment:
Interesting... in the past this feature was removed as being private tabs we would not want them to be restored... wondering if this is now in purpose again or a bug...

Reply:
There was a ticket for this functionality: https://mozilla-hub.atlassian.net/browse/FXIOS-7009

Reply:
Also there is a comment below this ticket saying we should be able to do that because is the same behaviour as on Android: https://mozilla-hub.atlassian.net/browse/FXIOS-2604